tests/contenttypes_tests/test_management.py

```diff
@@ -1,16 +1,23 @@
 from unittest import mock
 
 from django.apps.registry import Apps, apps
+from django.conf import settings
 from django.contrib.contenttypes import management as contenttypes_management
 from django.contrib.contenttypes.models import ContentType
 from django.core.management import call_command
 from django.test import TestCase
 from django.test.utils import captured_stdout
-
 from .models import ModelWithNullFKToSite, Post
 
 
 class UpdateContentTypesTests(TestCase):
+    available_apps = [
+        'contenttypes_tests',
+        'contenttypes_tests.app_with_no_models',
+        'django.contrib.contenttypes',
+    ]
+    apps.set_installed_apps(settings.INSTALLED_APPS + ['contenttypes_tests.app_with_no_models'])
```

Comment:
Could not really avoid putting those lines. The command that deletes the content types iterates over available apps. that's why I need the first line. "available_apps" needs to be a subset of "installed_apps", therefore a second line.

Is there a better way doing it?